exonum-java-binding/core/pom.xml

```diff
@@ -335,14 +370,6 @@
           <outputFile>${project.build.directory}/ejb-core-classpath.txt</outputFile>
           <includeScope>runtime</includeScope>
         </configuration>
-        <executions>
```

Comment:
If the execution is removed, this plugin configuration does not make sense and must be removed too. The file seems unused.

Reply:
I would fire an issue for replacing `maven-dependency-plugin` by `maven-assembly-plugin`

Reply:
What do you mean by replacing and what do you expect from assembly-plugin? There used to be a configuration to generate a classpath file that was used in scripts launching QA/cryptocurrency services. As the app creates a classpath from JARs in "lib/java", I don't see if anything is needed here (except copying the JARs).

Reply:
Agreed, removed.